contracts/SmartSession.sol

```diff
@@ -251,21 +253,28 @@ contract SmartSession is ISmartSession, SmartSessionBase, SmartSessionERC7739 {
         /*.•°:°.´+˚.*°.˚:*.´•*.+°.•°:´*.´•*.•°.•°:°.´:•˚°.*°.˚:*.´+°.•*/
         // Check UserOp policies
         // This reverts if policies are violated
-        vd = $userOpPolicies.check({
+        uint256 remainingPoliciesToCheck;
+        (vd, remainingPoliciesToCheck) = $userOpPolicies.check({
             userOp: userOp,
             permissionId: permissionId,
             callOnIPolicy: abi.encodeCall(IUserOpPolicy.checkUserOpPolicy, (permissionId.toConfigId(), userOp)),
             minPolicies: 0 // for userOp policies, a min of 0 is ok. since these are not security critical
          });
 
-        bytes4 selector = bytes4(userOp.callData[0:4]);
+        console2.log("policies enforced", remainingPoliciesToCheck);
+        // if no userOp policies were checked above,
+        // we must enforce, that at least one action policy is set for this PermissionId
+        remainingPoliciesToCheck = (remainingPoliciesToCheck == 0) ? 1 : 0;
```

Comment:
here,  from 'user op policies checked' it becomes 'min action policies' . so it changes the nature , that's why I decided to call it just `p`